lib/utils.js

```diff
@@ -32,8 +35,18 @@ function searchCache(moduleName, callback) {
   }
 }
 
+function makeProcessedInputPathAbsolutePath() {
+  var originalPath = this.serverless.config.serverless.processedInput.options.path
+  if (originalPath) {
+    var absolutePath = path.resolve(originalPath)
```

Comment:
`const` and semicolon.

Did you check this for a given absolute path (i.e. if originalPath is already absolute) too?